pkg/block/block_linux.go

```diff
@@ -237,6 +240,18 @@ func diskPartLabel(paths *linuxpath.Paths, disk string, partition string) string
 	return util.UNKNOWN
 }
 
+func diskPartTypeUdev(paths *linuxpath.Paths, disk string, partition string) string {
```

Comment:
Please add a quick docstring indicating this function is used as a fallback when `partitionInfo()` fails to determine a partition type, which happens when the partition is not mounted.